src/server/graphql/models/Task/rebalanceTask.js

```diff
@@ -0,0 +1,15 @@
+import r from 'server/database/rethinkDriver';
+import {STEP} from 'universal/utils/constants';
```

Comment:
I like this method. It's really cleaver.

`STEP` is a fairly ambiguous constant name, I'm going to rename it `SORT_STEP`.